### ISSUE TITLE
Stream Offline on fresh install (ansible)

### ISSUE THREAD

**Issue by Psychoses** [AzuraCast/AzuraCast#3754]
Hi,
***
**Using Docker, installation method and Host Operating System**

i don't use docker, my env is proxmox => lxc => ubuntu 20.04 => traditional ansible install
***
**AzuraCast version**

between  0.11.2 and 0.12
the 159991a work.
between 4ccddeb and 64423a7, i have an issue after install.
<s>between 64423a7 and 521feb4, i have an issue on install progress (Run AzuraCast Setup section)</s> FIXED !
<s>between 521feb4 and ac28dfb, the "Run AzuraCast Setup" error change a little.</s> FIXED !

***
**Describe the bug**

Can't fresh install any commit after 64423a7, i get :

<details>
  <summary>Click me to expand full error! <============= </summary>
  
```

TASK [azuracast-db-install : MariaDB Cleanup] **********************************
ok: [localhost] => (item=DELETE FROM mysql.user WHERE User='')
ok: [localhost] => (item=DELETE FROM mysql.user WHERE User='root' AND Host NOT IN ('localhost', '127.0.0.1', '::1'))
failed: [localhost] (item=DROP DATABASE test) => {"ansible_loop_var": "item", "changed": false, "cmd": ["mysql", "--defaults-extra-file=/root/.my.cnf", "-ne", "DROP DATABASE test"], "delta": "0:00:00.008910", "end": "2021-02-02 12:54:38.801369", "item": "DROP DATABASE test", "msg": "non-zero return code", "rc": 1, "start": "2021-02-02 12:54:38.792459", "stderr": "ERROR 1008 (HY000) at line 1: Can't drop database 'test'; database doesn't exist", "stderr_lines": ["ERROR 1008 (HY000) at line 1: Can't drop database 'test'; database doesn't exist"], "stdout": "", "stdout_lines": []}
ok: [localhost] => (item=DELETE FROM mysql.db WHERE Db='test' OR Db='test\_%')
...ignoring

TASK [azuracast-db-install : Set up AzuraCast database user] *******************
[WARNING]: Module did not set no_log for update_password
changed: [localhost]

TASK [azuracast-db-install : Add .my.cnf] **************************************
changed: [localhost]

TASK [azuracast-db-install : Reload privilege tables] **************************
ok: [localhost] => (item=FLUSH PRIVILEGES)

TASK [azuracast-db-install : Create MySQL Database] ****************************
changed: [localhost]

TASK [azuracast-db-install : Set up environment file] **************************
changed: [localhost] => (item={'option': 'MYSQL_HOST', 'value': 'azuracast'})
changed: [localhost] => (item={'option': 'MYSQL_PORT', 'value': '3306'})
changed: [localhost] => (item={'option': 'MYSQL_USER', 'value': 'azuracast'})
changed: [localhost] => (item={'option': 'MYSQL_DB', 'value': 'azuracast'})
changed: [localhost] => (item={'option': 'MYSQL_PASSWORD', 'value': 'mPxDh9Sy'})

TASK [ufw : Install UFW Firewall] **********************************************
ok: [localhost]

TASK [ufw : UFW - Disable all other incoming by default] ***********************
ok: [localhost]

TASK [ufw : UFW - Allow all other outgoing by default] *************************
ok: [localhost]

TASK [ufw : UFW - Enable AzuraCast TCP Ports] **********************************
changed: [localhost] => (item=80)
changed: [localhost] => (item=443)
changed: [localhost] => (item=8000:8999)
[WARNING]: The value 80 (type int) in a string field was converted to '80'
(type string). If this does not look like what you expect, quote the entire
value to ensure it does not change.
[WARNING]: The value 443 (type int) in a string field was converted to '443'
(type string). If this does not look like what you expect, quote the entire
value to ensure it does not change.

TASK [ufw : UFW - Enable SSH] **************************************************
changed: [localhost]

TASK [ufw : UFW - Turn on Firewall] ********************************************
changed: [localhost]

TASK [dbip : Download DBIP Database] *******************************************
changed: [localhost]

TASK [dbip : Extract DBIP Database] ********************************************
changed: [localhost]

TASK [dbip : Set DBIP Database Permissions] ************************************
changed: [localhost]

TASK [composer : Install Composer] *********************************************
changed: [localhost]

TASK [composer : Clear existing vendor folder] *********************************
changed: [localhost]

TASK [composer : Recreate vendor folder] ***************************************
changed: [localhost]

TASK [composer : Ensure composer.lock exists] **********************************
changed: [localhost]

TASK [composer : Install Composer Dependencies (Development)] ******************
skipping: [localhost]

TASK [composer : Install Composer Dependencies (Production)] *******************
changed: [localhost]

TASK [services : Enable and restart all core services] *************************
changed: [localhost] => (item=mysql)
changed: [localhost] => (item=php7.4-fpm)
changed: [localhost] => (item=nginx)
changed: [localhost] => (item=redis-server)
changed: [localhost] => (item=redis)

TASK [azuracast-build : Install NodeJS and NPM] ********************************
changed: [localhost]

TASK [azuracast-build : Set permissions for folders] ***************************
changed: [localhost] => (item=/var/azuracast/www/frontend/node_modules)
changed: [localhost] => (item=/var/azuracast/www/web/static)
changed: [localhost] => (item=/var/azuracast/www/web/static/dist)

TASK [azuracast-build : Pull Node Dependencies] ********************************
changed: [localhost]

TASK [azuracast-build : Build AzuraCast Frontend Scripts] **********************
changed: [localhost]

TASK [azuracast-build : Set permissions of generated frontend content] *********
changed: [localhost] => (item=chown -R azuracast:www-data /var/azuracast/www)
changed: [localhost] => (item=find /var/azuracast/www -type d -exec chmod 755 {} \;)
changed: [localhost] => (item=find /var/azuracast/www -type f -exec chmod 644 {} \;)
[WARNING]: Consider using the file module with owner rather than running
'chown'.  If you need to use command because file is insufficient you can add
'warn: false' to this command task or set 'command_warnings=False' in
ansible.cfg to get rid of this message.

TASK [azuracast-setup : Ensure update script continues to be executable] *******
changed: [localhost] => (item=/var/azuracast/www/update.sh)
changed: [localhost] => (item=/var/azuracast/www/bin/console)

TASK [azuracast-setup : Run AzuraCast Setup (Install Mode)] ********************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "php /var/azuracast/www/bin/console azuracast:setup", "delta": "0:00:00.150755", "end": "2021-02-02 12:57:01.034253", "msg": "non-zero return code", "rc": 255, "start": "2021-02-02 12:57:00.883498", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
```
</details>

some commit after between 521feb4 to ac28dfb the error change for : SQLSTATE php_network_getaddresses getaddrinfo failed on Running Database Migrations process

<details>
  <summary>Click me to expand full error! <============= </summary>
  
```

fatal: [localhost]: FAILED! => {"changed": true, "cmd": "php /var/azuracast/www/bin/console azuracast:setup", "delta": "0:00:00.408598", "end": "2021-01-28 20:42:56.711469", "msg": "non-zero return code", "rc": 1, "start": "2021-01-28 20:42:56.302871", "stderr": "[2021-01-28T20:42:56.699184+00:00] AzuraCast.ERROR: Doctrine\\DBAL\\Exception\\ConnectionException: An exception occurred in driver: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known (uncaught exception) at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 112 while running console command `azuracast:setup` {\"exception\":\"[object] (Doctrine\\\\DBAL\\\\Exception\\\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:112)\\n[previous exception] [object] (Doctrine\\\\DBAL\\\\Driver\\\\PDO\\\\Exception(code: 2002): SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\\n[previous exception] [object] (PDOException(code: 2002): SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:38)\\n[previous exception] [object] (PDOException(code: 0): PDO::__construct(): php_network_getaddresses: getaddrinfo failed: Name or service not known at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:38)\"} []\n[2021-01-28T20:42:56.700440+00:00] AzuraCast.WARNING: Console command `azuracast:setup` exited with error code 1. [] []\n\nIn AbstractMySQLDriver.php line 112:\n                                                                               \n  An exception occurred in driver: SQLSTATE[HY000] [2002] php_network_getaddr  \n  esses: getaddrinfo failed: Name or service not known                         \n                                                                               \n\nIn Exception.php line 18:\n                                                                               \n  SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name o  \n  r service not known                                                          \n                                                                               \n\nIn PDOConnection.php line 38:\n                                                                               \n  SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name o  \n  r service not known                                                          \n                                                                               \n\nIn PDOConnection.php line 38:\n                                                                               \n  PDO::__construct(): php_network_getaddresses: getaddrinfo failed: Name or s  \n  ervice not known                                                             \n                                                                               \n\nazuracast:setup [--update] [--load-fixtures] [--release]", "stderr_lines": ["[2021-01-28T20:42:56.699184+00:00] AzuraCast.ERROR: Doctrine\\DBAL\\Exception\\ConnectionException: An exception occurred in driver: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known (uncaught exception) at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 112 while running console command `azuracast:setup` {\"exception\":\"[object] (Doctrine\\\\DBAL\\\\Exception\\\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:112)\\n[previous exception] [object] (Doctrine\\\\DBAL\\\\Driver\\\\PDO\\\\Exception(code: 2002): SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18)\\n[previous exception] [object] (PDOException(code: 2002): SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:38)\\n[previous exception] [object] (PDOException(code: 0): PDO::__construct(): php_network_getaddresses: getaddrinfo failed: Name or service not known at /var/azuracast/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:38)\"} []", "[2021-01-28T20:42:56.700440+00:00] AzuraCast.WARNING: Console command `azuracast:setup` exited with error code 1. [] []", "", "In AbstractMySQLDriver.php line 112:", "                                                                               ", "  An exception occurred in driver: SQLSTATE[HY000] [2002] php_network_getaddr  ", "  esses: getaddrinfo failed: Name or service not known                         ", "                                                                               ", "", "In Exception.php line 18:", "                                                                               ", "  SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name o  ", "  r service not known                                                          ", "                                                                               ", "", "In PDOConnection.php line 38:", "                                                                               ", "  SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name o  ", "  r service not known                                                          ", "                                                                               ", "", "In PDOConnection.php line 38:", "                                                                               ", "  PDO::__construct(): php_network_getaddresses: getaddrinfo failed: Name or s  ", "  ervice not known                                                             ", "                                                                               ", "", "azuracast:setup [--update] [--load-fixtures] [--release]"], "stdout": "\nAzuraCast Setup\n===============\n\nWelcome to AzuraCast. Please wait while some key dependencies of AzuraCast are set up...\n * Environment: Production\n * Installation Method: Ansible\n\n\nRunning Database Migrations\n---------------------------", "stdout_lines": ["", "AzuraCast Setup", "===============", "", "Welcome to AzuraCast. Please wait while some key dependencies of AzuraCast are set up...", " * Environment: Production", " * Installation Method: Ansible", "", "", "Running Database Migrations", "---------------------------"]}

```
</details>

Some issues with php_network_getaddresses are related with disk space problem. It's not the case here while i can install previous commit without issue. So i suspect a problem with 64423a7 or 521feb4 and/or maybe 4ccddeb.

***

**Expected behavior**

Instant install success on fresh install.
Instead, i need to install 159991a and then update to ac28dfb for be up to date.

**Comment by SlvrEagle23** [comment]
@Psychoses I found the likely cause of this in the Ansible installation script, so if you run a new installation (with updated code, either by just following the setup instructions again or running `git reset --hard && git pull` in the folder), the installation should complete without any issue.

**Comment by Psychoses** [comment]
Hi @SlvrEagle23
Thx for the fix, i can now fresh install the new 67743922ec24b11799b8ce501a0b0801cbb18cf6 without ansible script error.

That fixed too this old issue #3558. I happy too because, you fix recently a many month old problem with the next song button. so the skip song button work now.

There is always a minor problem. Seems introduced after 4ccddeb. After install and setup the station show "Stream Offline" instead of generic azuracast "post install" audio message "Azuracast Is live !". I need to add song to the default playlist for get the stream up.

**Comment by SlvrEagle23** [comment]
@Psychoses That's likely a separate issue and it's possible we already have an issue on the books for that; please recreate this issue as a new one.
